packages/model-viewer/src/features/scene-graph/image.ts

```diff
@@ -75,16 +60,7 @@ export class Image extends ThreeDOMElement implements ImageInterface {
     });
 
     for (const texture of this[$threeTextures]) {
-      // If the URI is set to null but the Image had an associated buffer view
-      // (this would happen if it started out as embedded), then fall back to
-      // the cached object URL created by GLTFLoader:
-      if (image == null &&
-          (this[$sourceObject] as GLTFImage).bufferView != null) {
-        texture.image = this[$bufferViewImages].get(texture);
```

Comment:
@timmmeh As I've been going through this API, I realized that this complexity was confusing and uncalled-for. We never even specified that the input could be null, and it makes very little sense to go back to an original image, but only in certain circumstances. Anyway, hopefully this will simplify things for you as well.